labelbox/schema/project.py

```diff
@@ -840,6 +840,29 @@ def bulk_import_requests(self) -> PaginatedCollection:
                                    ["bulkImportRequests"],
                                    Entity.BulkImportRequest)
 
+    def batches(self) -> PaginatedCollection:
+        """ Fetch all batches that belong to this project
+
+        Returns:
+            A `PaginatedCollection of `Batch`es
+        """
+        id_param = "projectId"
+        query_str = """query GetProjectBatchesPyApi($from: String, $first: PageSize, $%s: ID!) {
+            project(where: {id: $%s}) {id
+            batches(after: $from, first: $first) { nodes { %s } pageInfo { endCursor }}}}
+        """ % (id_param, id_param, query.results_query_part(Entity.Batch))
+        return PaginatedCollection(
+            self.client,
+            query_str, {id_param: self.uid}, ['project', 'batches', 'nodes'],
+            lambda client, res: Entity.Batch(client, self.uid, res),
+            cursor_path={
+                'project': None,
+                'batches': None,
+                'pageInfo': None,
+                'endCursor': None
+            },
```

Comment:
nit: I think the type hint for PaginatedCollection is wrong. This can probably just be a list of the keys.

Reply:
Yeah this looked wronged to me too

Reply:
If you can also change the type hint from :
```cursor_path: Optional[Dict[str, Any]] = None```
to 
```cursor_path: Optional[List[str]] = None```
That would be awesome